src/Mariuzzo/LaravelJsLocalization/Generators/LangJsGenerator.php

```diff
@@ -214,4 +204,43 @@ private function getVendorKey($key)
 
         return $keyParts[2] .'.'. $keyParts[1] . '::' . $keyParts[3];
     }
+
+    private function processFile(&$messages, $file, $namespace = null)
+    {
+        $pathName = $file->getRelativePathName();
+        $extension = $file->getExtension();
+        if ($extension != 'php' && $extension != 'json') {
+            return;
+        }
+
+        if ($this->isMessagesExcluded($pathName)) {
+            return;
+        }
+
+        $key = substr($pathName, 0, -4);
+        $key = str_replace('\\', '.', $key);
+        $key = str_replace('/', '.', $key);
+
+        if ($namespace !== null)
+        {
+            $key = 'vendor.' . $namespace . '.'. $key;
+        }
+
+        if (Str::startsWith($key, 'vendor')) {
+            $key = $this->getVendorKey($key);
+        }
+
+        $fullPath = $file->getRealPath();
+        if ($extension == 'php') {
+            $messages[$key] = array_replace(Arr::get($messages,$key,[]), include $fullPath);
```

Comment:
I think you need to check the `include $fullPath` actually returns a filled array before adding it.